modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -923,6 +923,20 @@ The product's default behavior of using a cost-based IDP search algorithm when c
 For more information, see link:https://neo4j.com/docs/cypher-manual/current/query-tuning/query-options/#cypher-planner[Cypher manual -> Cypher planner].
 ====
 
+.Using the unicode \u0085 in an unescaped identifier
+====
+Query::
++
+[source,cypher]
+----
+RETURN 1 as my\u0085identifier
+----
+Description of the returned code::
+The unicode `\\u0061` is deprecated in unescaped identifiers.
+Suggestions for improvement::
+If you want to continue to use this unicode, please escape the identifier instead.
```

Comment:
```suggestion
Description of the returned code::
The unicode `\\u0061` is deprecated in unescaped identifiers.
If you want to continue to use this unicode, please escape the identifier instead.
```

I know this is not super clear in the instruction, 'Description of the returned code' is supposed to be the same description as in Neo4j, and 'Suggestions for improvement' is any other suggestion which you might want to provide but doesn't fit in the description (e.g. too long text or need an example).